www-servers/sniproxy/sniproxy-0.5.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/dlundquist/sniproxy.git"
+	EGIT_BRANCH="master"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/dlundquist/sniproxy/archive/${PV}.tar.gz -> sniproxy-${PV}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Proxies incoming HTTP and TLS connections based on the hostname."
+HOMEPAGE="https://github.com/dlundquist/sniproxy"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="+dns rfc3339 +largefile"
+
+RDEPEND="
+	dev-libs/libev
+	>=dev-libs/libpcre-3
+	dns? ( net-libs/udns )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+	./autogen.sh || die "Autogen failed"
+
+	sed -i "s/ daemon/ sniproxy/" debian/sniproxy.conf debian/logrotate.conf || die "Unable to replace configuration"
+}
+
+src_install() {
+	default
+
+	newinitd "${FILESDIR}/sniproxy.init" ${PN}
+
+	insinto /etc/sniproxy
+	newins "${S}"/debian/sniproxy.conf sniproxy.conf
+
+	keepdir /var/log/${PN}
+
+	insinto /etc/logrotate.d
+	newins "${S}"/debian/logrotate.conf ${PN}
+
+	dodoc ARCHITECTURE.md AUTHORS README.md
+	doman man/${PN}.8
```

Comment:
Avoid putting `${PN}` all over `src_install()`. It's unreadable, especially when you're referring to paths you don't control yourself.